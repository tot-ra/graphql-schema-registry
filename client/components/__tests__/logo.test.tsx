import { render } from '@testing-library/react';
import Logo from '../logo';

describe('tests for Logo', () => {
	it('should render and match snapshot', () => {
		const { container } = render(<Logo />);

		expect(container).toMatchSnapshot();
	});
});
