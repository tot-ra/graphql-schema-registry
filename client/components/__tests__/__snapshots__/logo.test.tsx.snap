// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests for Logo should render and match snapshot 1`] = `
<div>
  <div>
    <svg
      height="45"
      preserveAspectRatio="xMidYMid meet"
      version="1.1"
      viewBox="0 0 640 640"
      width="45"
    >
      <defs>
        <path
          d="M0 0L640 0L640 640L0 640L0 0Z"
          id="aACFKWqId"
        />
        <path
          d="M90.39 455.37L309.93 68.35"
          id="abQRk6fPB"
        />
        <path
          d="M312.15 593.54L539.33 180.06"
          id="c43cxpRdDt"
        />
        <path
          d="M315.05 324.61L90.39 174.35"
          id="e1RwQ287qq"
        />
        <path
          d="M537.24 450.83L312.15 321.52"
          id="aOTTscD3s"
        />
        <path
          d="M539.33 173.42L312.15 66.26"
          id="a5hX0shUV9"
        />
        <path
          d="M312.15 596.86L312.15 328.8"
          id="f4aRsobM8F"
        />
        <path
          d="M312.15 324.61L91.71 450.83"
          id="d4df3gG5KI"
        />
        <path
          d="M90.39 176.17L309.93 66.26"
          id="a3f1xiOUe"
        />
        <path
          d="M539.33 450.83L539.33 180.06"
          id="j2kppAtKaC"
        />
        <path
          d="M312.15 596.86L539.33 450.83"
          id="fmMlTgAod"
        />
        <path
          d="M90.39 450.83L90.39 176.17"
          id="a4gSTNVKSn"
        />
        <path
          d="M312.15 593.54L90.39 455.37"
          id="b3WdzhfVx"
        />
        <path
          d="M360.8 63.37C360.8 89.94 339 111.51 312.15 111.51C285.3 111.51 263.5 89.94 263.5 63.37C263.5 36.8 285.3 15.22 312.15 15.22C339 15.22 360.8 36.8 360.8 63.37Z"
          id="b5rcw9cXTN"
        />
        <path
          d="M300.64 63.37L288.94 52.03L312.34 52.03L335.74 52.03L324.04 63.37L312.34 74.7L300.64 63.37Z"
          id="c1R0nOtLXZ"
        />
        <path
          d="M360.99 593.54C360.99 620.11 339.19 641.69 312.34 641.69C285.49 641.69 263.7 620.11 263.7 593.54C263.7 566.97 285.49 545.4 312.34 545.4C339.19 545.4 360.99 566.97 360.99 593.54Z"
          id="g59tMZda0"
        />
        <path
          d="M288.75 591.21L288.75 587.03L335.55 587.03L335.55 591.21L288.75 591.21Z"
          id="a9tGUK1OE"
        />
        <path
          d="M288.75 601.04L288.75 596.86L335.55 596.86L335.55 601.04L288.75 601.04Z"
          id="c5b12KQzOZ"
        />
        <path
          d="M585.89 176.17C585.89 202.74 564.09 224.31 537.24 224.31C510.39 224.31 488.59 202.74 488.59 176.17C488.59 149.59 510.39 128.02 537.24 128.02C564.09 128.02 585.89 149.59 585.89 176.17Z"
          id="baUtSPS7w"
        />
        <path
          d="M562.73 187.5L562.73 191.68L515.93 191.68L515.93 187.5L562.73 187.5Z"
          id="i125F4EsAe"
        />
        <path
          d="M285.75 67.55L285.75 63.37L339.81 63.37L339.81 67.55L285.75 67.55Z"
          id="a1nj584xmL"
        />
        <path
          d="M285.12 60.89L285.12 56.7L339.18 56.7L339.18 60.89L285.12 60.89Z"
          id="a6JHlaw3Z7"
        />
        <path
          d="M527.63 176.17L515.93 164.83L539.33 164.83L562.73 164.83L551.03 176.17L539.33 187.5L527.63 176.17Z"
          id="hE0r2Q1TQ"
        />
        <path
          d="M139.04 176.17C139.04 202.74 117.24 224.31 90.39 224.31C63.54 224.31 41.75 202.74 41.75 176.17C41.75 149.59 63.54 128.02 90.39 128.02C117.24 128.02 139.04 149.59 139.04 176.17Z"
          id="iDavGLfO"
        />
        <path
          d="M587.98 450.83C587.98 477.41 566.18 498.98 539.33 498.98C512.48 498.98 490.69 477.41 490.69 450.83C490.69 424.26 512.48 402.69 539.33 402.69C566.18 402.69 587.98 424.26 587.98 450.83Z"
          id="jPMSGJE9O"
        />
        <path
          d="M68.31 173.42L68.31 169.24L115.11 169.24L115.11 173.42L68.31 173.42Z"
          id="b5F6Ihm6"
        />
        <path
          d="M73.45 192.44L70.83 189.18L107.34 159.89L109.96 163.16L73.45 192.44Z"
          id="b2Hg1A9GwM"
        />
        <path
          d="M68.31 184.24L68.31 180.06L115.11 180.06L115.11 184.24L68.31 184.24Z"
          id="cqhQeCAhb"
        />
        <path
          d="M140.36 450.83C140.36 477.41 118.56 498.98 91.71 498.98C64.86 498.98 43.06 477.41 43.06 450.83C43.06 424.26 64.86 402.69 91.71 402.69C118.56 402.69 140.36 424.26 140.36 450.83Z"
          id="aalbRYX86"
        />
        <path
          d="M549.87 427.43L554.05 427.43L554.05 474.23L549.87 474.23L549.87 427.43Z"
          id="am8voaChl"
        />
        <path
          d="M362.58 324.61C362.58 351.19 340.78 372.76 313.93 372.76C287.09 372.76 265.29 351.19 265.29 324.61C265.29 298.04 287.09 276.47 313.93 276.47C340.78 276.47 362.58 298.04 362.58 324.61Z"
          id="a14pdItL6b"
        />
        <path
          d="M524.61 431.53L549.87 431.53L549.87 470.47L524.61 470.47L524.61 431.53Z"
          id="b7JJyZDlAE"
        />
        <path
          d="M101.12 432.09C101.12 437.28 96.9 441.5 91.71 441.5C86.52 441.5 82.31 437.28 82.31 432.09C82.31 426.9 86.52 422.69 91.71 422.69C96.9 422.69 101.12 426.9 101.12 432.09Z"
          id="fuiZXz5Im"
        />
        <path
          d="M94.26 459.96C89.26 461.36 84.06 458.45 82.66 453.45C81.25 448.45 84.17 443.26 89.16 441.85C94.16 440.44 99.36 443.36 100.76 448.35C102.17 453.35 99.25 458.55 94.26 459.96Z"
          id="a115Mh34n6"
        />
        <path
          d="M101.12 469.58C101.12 474.77 96.9 478.98 91.71 478.98C86.52 478.98 82.31 474.77 82.31 469.58C82.31 464.38 86.52 460.17 91.71 460.17C96.9 460.17 101.12 464.38 101.12 469.58Z"
          id="a4pBNc1nkM"
        />
        <path
          d="M328.07 321.52L341.05 334.09L315.09 334.09L289.13 334.09L302.11 321.52L315.09 308.94L328.07 321.52Z"
          id="d2erpfESzn"
        />
        <path
          d="M298.5 320.17L298.5 315.99L335.55 315.99L335.55 320.17L298.5 320.17Z"
          id="a2Wf92zTkv"
        />
        <path
          d="M286.9 328.8L286.9 324.61L340.96 324.61L340.96 328.8L286.9 328.8Z"
          id="cuV9Yd1bo"
        />
      </defs>
      <g>
        <g>
          <g>
            <use
              fill="green"
              fill-opacity="1"
              href="#aACFKWqId"
              opacity="1"
            />
          </g>
          <g>
            <g>
              <use
                fill-opacity="0"
                href="#abQRk6fPB"
                opacity="1"
                stroke="white"
                stroke-opacity="1"
                stroke-width="25"
              />
            </g>
          </g>
          <g>
            <g>
              <use
                fill-opacity="0"
                href="#c43cxpRdDt"
                opacity="1"
                stroke="white"
                stroke-opacity="1"
                stroke-width="25"
              />
            </g>
          </g>
          <g>
            <g>
              <use
                fill-opacity="0"
                href="#e1RwQ287qq"
                opacity="1"
                stroke="white"
                stroke-opacity="1"
                stroke-width="25"
              />
            </g>
          </g>
          <g>
            <g>
              <use
                fill-opacity="0"
                href="#aOTTscD3s"
                opacity="1"
                stroke="white"
                stroke-opacity="1"
                stroke-width="25"
              />
            </g>
          </g>
          <g>
            <g>
              <use
                fill-opacity="0"
                href="#a5hX0shUV9"
                opacity="1"
                stroke="white"
                stroke-opacity="1"
                stroke-width="30"
              />
            </g>
          </g>
          <g>
            <g>
              <use
                fill-opacity="0"
                href="#f4aRsobM8F"
                opacity="1"
                stroke="white"
                stroke-opacity="1"
                stroke-width="25"
              />
            </g>
          </g>
          <g>
            <g>
              <use
                fill-opacity="0"
                href="#d4df3gG5KI"
                opacity="1"
                stroke="white"
                stroke-opacity="1"
                stroke-width="25"
              />
            </g>
          </g>
          <g>
            <g>
              <use
                fill-opacity="0"
                href="#a3f1xiOUe"
                opacity="1"
                stroke="white"
                stroke-opacity="1"
                stroke-width="30"
              />
            </g>
          </g>
          <g>
            <g>
              <use
                fill-opacity="0"
                href="#j2kppAtKaC"
                opacity="1"
                stroke="white"
                stroke-opacity="1"
                stroke-width="30"
              />
            </g>
          </g>
          <g>
            <g>
              <use
                fill-opacity="0"
                href="#fmMlTgAod"
                opacity="1"
                stroke="white"
                stroke-opacity="1"
                stroke-width="30"
              />
            </g>
          </g>
          <g>
            <g>
              <use
                fill-opacity="0"
                href="#a4gSTNVKSn"
                opacity="1"
                stroke="white"
                stroke-opacity="1"
                stroke-width="30"
              />
            </g>
          </g>
          <g>
            <g>
              <use
                fill-opacity="0"
                href="#b3WdzhfVx"
                opacity="1"
                stroke="white"
                stroke-opacity="1"
                stroke-width="30"
              />
            </g>
          </g>
          <g>
            <use
              fill="#0a8400"
              fill-opacity="1"
              href="#b5rcw9cXTN"
              opacity="1"
            />
          </g>
          <g>
            <use
              fill="#ffffff"
              fill-opacity="1"
              href="#c1R0nOtLXZ"
              opacity="1"
            />
          </g>
          <g>
            <use
              fill="#0a8400"
              fill-opacity="1"
              href="#g59tMZda0"
              opacity="1"
            />
          </g>
          <g>
            <use
              fill="#ffffff"
              fill-opacity="1"
              href="#a9tGUK1OE"
              opacity="1"
            />
          </g>
          <g>
            <use
              fill="#ffffff"
              fill-opacity="1"
              href="#c5b12KQzOZ"
              opacity="1"
            />
          </g>
          <g>
            <use
              fill="#0a8400"
              fill-opacity="1"
              href="#baUtSPS7w"
              opacity="1"
            />
          </g>
          <g>
            <use
              fill="#ffffff"
              fill-opacity="1"
              href="#i125F4EsAe"
              opacity="1"
            />
          </g>
          <g>
            <use
              fill="#0a8400"
              fill-opacity="1"
              href="#a1nj584xmL"
              opacity="1"
            />
          </g>
          <g>
            <use
              fill="#0a8400"
              fill-opacity="1"
              href="#a6JHlaw3Z7"
              opacity="1"
            />
          </g>
          <g>
            <use
              fill="#ffffff"
              fill-opacity="1"
              href="#hE0r2Q1TQ"
              opacity="1"
            />
          </g>
          <g>
            <use
              fill="#0a8400"
              fill-opacity="1"
              href="#iDavGLfO"
              opacity="1"
            />
          </g>
          <g>
            <use
              fill="#0a8400"
              fill-opacity="1"
              href="#jPMSGJE9O"
              opacity="1"
            />
          </g>
          <g>
            <use
              fill="#ffffff"
              fill-opacity="1"
              href="#b5F6Ihm6"
              opacity="1"
            />
          </g>
          <g>
            <use
              fill="#ffffff"
              fill-opacity="1"
              href="#b2Hg1A9GwM"
              opacity="1"
            />
          </g>
          <g>
            <use
              fill="#ffffff"
              fill-opacity="1"
              href="#cqhQeCAhb"
              opacity="1"
            />
          </g>
          <g>
            <use
              fill="#0a8400"
              fill-opacity="1"
              href="#aalbRYX86"
              opacity="1"
            />
          </g>
          <g>
            <use
              fill="#ffffff"
              fill-opacity="1"
              href="#am8voaChl"
              opacity="1"
            />
          </g>
          <g>
            <use
              fill="#0a8400"
              fill-opacity="1"
              href="#a14pdItL6b"
              opacity="1"
            />
          </g>
          <g>
            <use
              fill="#ffffff"
              fill-opacity="1"
              href="#b7JJyZDlAE"
              opacity="1"
            />
          </g>
          <g>
            <use
              fill="#ffffff"
              fill-opacity="1"
              href="#fuiZXz5Im"
              opacity="1"
            />
          </g>
          <g>
            <use
              fill="#ffffff"
              fill-opacity="1"
              href="#a115Mh34n6"
              opacity="1"
            />
          </g>
          <g>
            <use
              fill="#ffffff"
              fill-opacity="1"
              href="#a4pBNc1nkM"
              opacity="1"
            />
          </g>
          <g>
            <use
              fill="#ffffff"
              fill-opacity="1"
              href="#d2erpfESzn"
              opacity="1"
            />
          </g>
          <g>
            <use
              fill="#0a8400"
              fill-opacity="1"
              href="#a2Wf92zTkv"
              opacity="1"
            />
          </g>
          <g>
            <use
              fill="#0a8400"
              fill-opacity="1"
              href="#cuV9Yd1bo"
              opacity="1"
            />
          </g>
        </g>
      </g>
    </svg>
  </div>
</div>
`;
